golang/book-go-modern-prod-ready-services/01-install/http-handler.go

```
@@ -0,0 +1,21 @@
+
+func HomeHandler(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "Welcome to the home page!")
+}
+
+func main() {
+	router := mux.NewRouter()
+	router.HandleFunc("/", HomeHandler)
+	http.Handle("/", router)
+
+	fmt.Println("Starting server on port 8080")
+	http.ListenAndServe(":8080", router)
```

golangci-lint (1.62.2)

Error return value of `http.ListenAndServe` is not checked

(errcheck)